### PR TITLE
US1E

### LINKED ISSUE #12: US1E: As a user, I want to be able to view additional information about campus buildings, directly, by clicking on them.
Additional information should the abbreviated name, opening hours, accessibility options, departments and available stores (library Bookstore, Tim Hortons, etc).

### LINKED ISSUE #84: US1E - Make polygons separately touchable

### LINKED ISSUE #85: US1E - Retrieve and add information about each building to 'buildings' array

### LINKED ISSUE #86: US1E - Present building information in pop-up after pressing on a building

### PR DESCRIPTION
**Pull request by TalalBazerbachi**
- Preview building information when tapped on.

### CODE REVIEW

File: components/addresses/index.js [hunk 1]
```diff
@@ -28,69 +28,70 @@ export default class Addresses extends Component {
     };
   }
 
-  drawPath =() => {
-    this.setState((prevState) => { return { drawPath: !prevState.drawPath }; });
-  }
+  drawPath = () => {
```
Comment (gisellemartel) [inline components/addresses/index.js:35]: Don't forget to add JSDoc comment for each function

**Review by gisellemartel — CHANGES_REQUESTED** [review]
Overall very good! Feature functionality works as intended and code looks clean. The only issue is that there seems to be a lag in the rendering of the photos in the carousel, which looks strange since there is a visible lag between the rendering of the indoor map and the carousel. I would suggest to render both at the same time

File: components/home/index.js [hunk 2]
```diff
@@ -32,17 +34,13 @@ class Home extends Component {
       },
       interiorMode: false,
       nearbyMarkers: [],
-      isVisible: false,
       // eslint-disable-next-line react/no-unused-state
       isSearchVisible: true,
-      isGoVisible: false,
-      isSwitchAvailableIndestination: true,
-
       // current Concordia a user is in
```
Comment (eufekt) [inline components/home/index.js]: please be more specific. What is Concordia ?

File: components/switchCampuses/styles.js [hunk 3]
```diff
@@ -1,83 +1,81 @@
...
-    borderStyle: 'solid',
+    borderStyle: "solid",
     borderWidth: 1,
-    borderColor: 'black',
-    backgroundColor: '#ffffff',
-    justifyContent: 'center',
-    alignItems: 'center',
-
+    borderColor: "black",
+    backgroundColor: "#ffffff",
```
Comment (eufekt) [inline components/switchCampuses/styles.js]: my linter on my computer wants me to use single quote. Does your eslint.json comes from master  ?

**Review by trevornag — CHANGES_REQUESTED** [review]
Please fix conflicts as well

**Review by eufekt — COMMENTED** [review]
Please fix conflicts with master. Functionality looks good. One thing to keep in mind is that all the pictures of the school are making the app heavier. Perhaps we should redue their quality so that they are all 300kb each MAX. I have seen a few that are over 1mb. This is not a priority but will make better performance for loading as well. Please also address comments. Good refactoring and simplification of CustomPolygon component.

File: components/addresses/index.js (hunk 1 above)
Comment (mohhef) [reply components/addresses/index.js:35]: Fixed everything, except having images load at the same time as the component. Would you have any insight on how is this achievable?

File: components/home/index.js (hunk 2 above)
Comment (mohhef) [reply components/home/index.js]: Done

File: components/switchCampuses/styles.js (hunk 3 above)
Comment (mohhef) [reply components/switchCampuses/styles.js]: Done

File: components/addresses/index.js (hunk 1 above)
Comment (eufekt) [reply components/addresses/index.js:35]: maybe decrease image size to <200kb each

**Comment by eufekt** [comment]
Comments I addressed have been resolved.

**Review by eufekt — APPROVED** [review]
Comments I addressed have been resolved

File: components/addresses/index.js (hunk 1 above)
Comment (mohhef) [reply components/addresses/index.js:35]: Reduced Overall image sizes from 32.4 mb to 10.6

**Review by gisellemartel — APPROVED** [review]

**Review by h-karim — APPROVED** [review]
Looks good:
- checking the other comments all issues seem to be resolved                  
- the features work well locally.

**Review by trevornag — APPROVED** [review]

**Pull request merged**